osidb/mixins.py

```diff
@@ -59,6 +59,32 @@ def save(self, *args, auto_timestamps=True, **kwargs):
         super().save(*args, **kwargs)
 
 
+class TrackingMixinManager(models.Manager):
```

Comment:
I am kinda confused by this manager, can you give me a quick explanation please?

Reply:
Ahhh, now I get it, thank you!